WalletWasabi.Tests/UnitTests/Crypto/GroupElementTests.cs

```diff
@@ -274,5 +274,155 @@ public void Negation()
 			Assert.Equal(new GroupElement(EC.G.Negate() * one), new GroupElement(EC.G * one).Negate());
 			Assert.Equal(GroupElement.Infinity, GroupElement.Infinity.Negate());
 		}
+
+		private byte[] FillByteArray(int length, byte character)
+		{
+			var array = new byte[length];
+			for (int i = 0; i < array.Length; i++)
+			{
+				array[i] = character;
+			}
```

Comment:
It is possible to use:

```suggestion
			Array.Fill(array, character);
```

Reply:
https://source.dot.net/#System.Private.CoreLib/Array.cs,8e23f7b1df2406cc